### PR TITLE
Highlights whole row

### PR DESCRIPTION
**Pull request by rjs68**
Removed hasFocus from cell rendering
Replaced with property to add border if the row is equal to the selected row
Added repaint in getSelectedRow() to ensure multiple rows aren't highlighted at once

### CODE REVIEW

**Review by Arden488 — APPROVED** [review]

**Review by BelfDev — CHANGES_REQUESTED** [review]
Screen Shot 2020-03-05 at 11 12 18 AM

File: src/main/java/madstax/view/PlannerCellRenderer.java [hunk 1]
```diff
@@ -4,17 +4,23 @@
...
 import javax.swing.*;
+import javax.swing.border.MatteBorder;
 import javax.swing.table.DefaultTableCellRenderer;
 import javax.swing.table.TableCellRenderer;
 import java.awt.*;
 import java.util.ArrayList;
 
 @SuppressWarnings("unchecked")
 public class PlannerCellRenderer extends JLabel implements TableCellRenderer {
+    private int selectedRow;
```
Comment (BelfDev) [inline src/main/java/madstax/view/PlannerCellRenderer.java]: The table has its own API to tell you which row is currently selected.

File: src/main/java/madstax/view/screen/CoursePlannerScreen.java [hunk 2]
```diff
@@ -101,6 +101,7 @@ public void selectFirstRow() {
     }
 
     public int getSelectedRow() {
+        table.repaint();
```
Comment (BelfDev) [inline src/main/java/madstax/view/screen/CoursePlannerScreen.java]: This looks a bit weird!

**Review by BelfDev — CHANGES_REQUESTED** [review]

File: src/main/java/madstax/view/PlannerCellRenderer.java (hunk 1 above)
Comment (rjs68) [reply src/main/java/madstax/view/PlannerCellRenderer.java]: I fixed this - it helped with the issue of selecting 2 rows that you mentioned as well!

File: src/main/java/madstax/view/screen/CoursePlannerScreen.java (hunk 2 above)
Comment (rjs68) [reply src/main/java/madstax/view/screen/CoursePlannerScreen.java]: I agree! But it was the only way I could think to stop it selecting multiple rows

Comment (rjs68) [reply src/main/java/madstax/view/screen/CoursePlannerScreen.java]: Actually ignore this - I can remove it now due to the change in how I get the selected row!

**Review by BelfDev — APPROVED** [review]

**Pull request merged**